runtime/src/lib.rs

```diff
@@ -778,6 +805,7 @@ construct_runtime!(
         Bounties: pallet_bounties,
         ChildBounties: pallet_child_bounties,
         Utility: pallet_utility,
+        Proxy: pallet_proxy,
```

Comment:
Till we don't use the index is better to add new pallets at the end of the list.